src/ScratchStorage.js

```diff
@@ -154,7 +169,7 @@ class ScratchStorage {
                                 } else {
                                     // TODO? this.localHelper.cache(assetType, assetId, asset);
                                     if (helper !== this.builtinHelper && assetType.immutable) {
-                                        asset.assetId = this.builtinHelper.cache(
+                                        asset.assetId = this.builtinHelper._store(
```

Comment:
In a follow-up we should add a breaking change and remove this behavior altogether.